src/components/MultiHopTrade/components/TradeInput/components/TradeQuotes/TradeQuote.tsx

```diff
@@ -175,17 +157,20 @@ export const TradeQuoteLoaded: FC<TradeQuoteProps> = ({
   const activeSwapperColor = (() => {
     if (!isTradingActive) return redColor
     if (!hasAmountWithPositiveReceive) return redColor
-    if (isActive) return greenColor
+    if (isActive) return 'border.focused'
     return borderColor
   })()
 
   const hoverProps = useMemo(
-    () => ({ borderColor: isActive ? activeSwapperColor : hoverColor, cursor: 'pointer' }),
-    [activeSwapperColor, hoverColor, isActive],
+    () => ({
+      cursor: 'pointer',
+      bg: 'background.surface.hover',
+    }),
+    [],
```

Comment:
this can be moved to a hardcoded object at the top now that there are no dynamic properties